elements/pfe-content-set/demo/index.html

```diff
@@ -1,328 +1,385 @@
-    
-    <link rel="stylesheet" href="../../pfe-styles/dist/pfe-base.css" />
-    <link rel="stylesheet" href="../../pfe-styles/dist/pfe-context.css" />
-    <link rel="stylesheet" href="../../pfe-styles/dist/pfe-layouts.css" />
-
-    <noscript>
-      <link href="../../pfelement/dist/pfelement--noscript.min.css" rel="stylesheet">
-    </noscript>
-
-    <link href="../../pfelement/dist/pfelement.min.css" rel="stylesheet">
-
-    <!-- uncomment the es5-adapter if you're using the umd version -->
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.3.0/custom-elements-es5-adapter.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.3.0/webcomponents-bundle.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.min.js"></script>
-    <script>
-      require([
-        "../dist/pfe-content-set.umd.js",
-        "../../pfe-cta/dist/pfe-cta.umd.js"
-      ])
-    </script>
-
-    <noscript>
-      <link href="../../pfelement/dist/pfelement--noscript.min.css" rel="stylesheet">
-    </noscript>
-    <style>
-      .faux-container {
-        border: 1px dashed #ccc;
-        padding: 30px;
-      }
-      h2 {
-        font-size: 1.2em;
-        color: var(--pfe-broadcasted--text);
-      }
-      h3 {
-        font-size: 1.4em;
-        color: var(--pfe-broadcasted--text);
-      }
-      hr {
-        margin: 40px 10px;
-        border-color: transparent;
-        border-bottom-color: #ccc;
-      }
-
-      section {
-        padding: 2rem;
-      }
-
-      .dark-background {
-        background-color: #000;
-      }
-
-      .border-bottom {
-        border-bottom: 1px solid var(--pfe-broadcasted--text);
-      }
-    </style>
-  </head>
-  <body unresolved>
+
+  <link rel="stylesheet" href="../../pfe-styles/dist/pfe-base.css" />
+  <link rel="stylesheet" href="../../pfe-styles/dist/pfe-context.css" />
+  <link rel="stylesheet" href="../../pfe-styles/dist/pfe-layouts.css" />
+
+  <noscript>
+    <link href="../../pfelement/dist/pfelement--noscript.min.css" rel="stylesheet">
+  </noscript>
+
+  <link href="../../pfelement/dist/pfelement.min.css" rel="stylesheet">
+
+  <!-- uncomment the es5-adapter if you're using the umd version -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.3.0/custom-elements-es5-adapter.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/2.3.0/webcomponents-bundle.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.min.js"></script>
+  <script>
+    require([
+      "../dist/pfe-content-set.umd.js",
+      "../../pfe-cta/dist/pfe-cta.umd.js",
+      "../../pfe-band/dist/pfe-band.umd.js"
+    ])
+
+  </script>
+
+  <noscript>
+    <link href="../../pfelement/dist/pfelement--noscript.min.css" rel="stylesheet">
+    <link href="https://static.redhat.com/libs/redhat/redhat-theme/latest/advanced-theme.css" rel="stylesheet">
```

Comment:
I think the theme would be loaded outside the noscript tag

Reply:
thx! fixed